app/src/main/java/org/mifos/selfserviceapp/ui/activities/SavingsAccountContainerActivity.java

```diff
@@ -18,14 +18,15 @@
 public class SavingsAccountContainerActivity extends BaseActivity {
 
     private long savingsId;
+    private long accountId;
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_container);
-
         savingsId = getIntent().getExtras().getLong(Constants.SAVINGS_ID);
-
-        replaceFragment(SavingAccountsDetailFragment.newInstance(savingsId), false, R.id.container);
+        accountId = getIntent().getExtras().getLong(Constants.ACCOUNT_ID);
+        replaceFragment(SavingAccountsDetailFragment.newInstance(savingsId, accountId), 
+                false, R.id.container);
```

Comment:
Why you are passing Savings Accounts Id twice. Please revert back these changes.